frontend/src/components/AdminDashboard/UserCheckIn.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from "react";
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { UsersCheckIn } from "@/interfaces/IAnalytics";
+import { Checkbox } from "../ui/checkbox";
+import { useNavigate } from "react-router-dom";
+import useAnalyticsApi from "@/hooks/api/useAnalyticsApi";
+import Loader from "../ui/Loader";
+import { toast } from "sonner";
+import { ERROR_MESSAGES } from "@/enums/ErrorMessages";
+
+const UserCheckIn = () => {
+  const navigate = useNavigate();
+  const { getAllCheckInUsers, checkOffUser } = useAnalyticsApi();
+
+  const [users, setUsers] = useState<UsersCheckIn[] | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleCheckChange = (id: string) => {
+    if (!users) return;
+    checkOffUser(id)
+      .then(() => toast.success(`משתמש סומן בהצלחה!`))
+      .catch((err) =>
+        toast.error(ERROR_MESSAGES.GENERIC_ERROR_MESSAGE, {
+          description: err.response.data.message,
+        })
+      );
+
+    setTimeout(() => {
+      const newUsers = users.filter((user) => user._id !== id);
+
+      setUsers(newUsers);
+    }, 800);
+  };
+
+  useEffect(() => {
+    setIsLoading(true);
+    getAllCheckInUsers()
+      .then((res) => setUsers(res))
+      .catch((err) => console.log(err))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  return (
+    <Card dir="rtl" className="w-[40%] shadow-md py-5">
```

Comment:
when i make my screen smaller the 40% squeezes the list so it doesn't look good. can you make it responsive? Use tailwind media queries.